CHANGELOG.rst

```diff
@@ -1,6 +1,72 @@
 Change Log
 ==========
 
+Unreleased
+----------
+
+* silx view application:
+
+  * Added: keep the same axes selection when changing dataset except for the stack view (PR #2701, #2780)
+  * Added a Description column in the browsing tree to display NeXus title or name (PR #2804)
+  * Added support of URL as filename (PR #2750)
+
+* `silx.gui`:
+
+  * `silx.gui.plot`:
+
+    * Added scatter plot regular and irregular grid visualization mode (PR #2810, #2815, #2820, #2824, #2831)
+    * Added `baseline` argument to `PlotWidget` `addCurve` and `addHistogram` methods (PR #2715)
+    * Added right axis support to `PlotWidget` marker items (PR #2744)
+    * Added `BoundingRect` `PlotWidget` item (PR #2823)
+    * Added more markers to `PlotWidget` items using symbols (PR #2792)
+    * Improved and fixed `PlotWidget` and backends rendering and picking to guarantee rendering order of items (PR #2602, #2694, #2726, #2728, #2730, #2731, #2732, #2734, #2746, #2800, #2822, #2829)
+  * Improved `RegionOfInterest`: Added `sigItemChanged` signal, renamed `get|setLabel` to `get|setName` (PR #2684, #2729, #2794, #2803)
+  * Improved `StackView`: Allow to save dataset to HDF5 (PR #2813)
+
+  * `silx.gui.plot3d`:
+
+    * Added colormapped isosurface display to `ComplexField3D` (PR #2675)
+
+  * Miscellaneous:
+
+    * Added `cividis` colormap (PR #2763)
+    * Added `silx.gui.widgets.ColormapNameComboBox` widget (PR #2814)
+    * Added `silx.gui.widgets.LegendIconWidget` widget (PR #2783)
+    * Added `silx.gui.utils.blockSignals` context manager (PR #2697, #2702)
+    * Added `silx.gui.utils.qtutils.getQEventName` function (PR #2725)
+    * Added `silx.gui.colors.asQColor` function (PR #2753)
+    * Minor fixes (PR #2662, #2667, #2674, #2719, #2724, #2747, #2757, #2760, #2766, #2789, #2798, #2799, #2805, #2811)
+
+* `silx.opencl`:
+
+  * Improved `silx.opencl.sparse.CSR` to support more data types (PR #2671)
```

Comment:
The sparse.CSR feature was not announced in `0.11.0`, perhaps it is time now to advertise it ?

Reply:
If it's ready yes!